commons-vfs2/src/main/java/org/apache/commons/vfs2/operations/vcs/package-info.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * VFS version control operations.
+ *
+ * <p>
+ *  This package contains common interfaces for Version Control Systems such as
+ *  SVN, CVS, etc.
```

Comment:
But not Git?

Reply:
I just copied over the old docs.

We can add Git, would you be happy with the following?

```suggestion
 *  Git, SVN, CVS, etc.
```

Reply:
Sure, TY.